SmartDeviceLink/SDLLifecycleConfigurationUpdate.h

```diff
@@ -0,0 +1,35 @@
+//
+//  SDLLifecycleConfigurationUpdate.h
+//  SmartDeviceLink-iOS
+//
+//  Created by Kujtim Shala on 06.09.17.
+//  Copyright © 2017 smartdevicelink. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class SDLTTSChunk;
+
+@interface SDLLifecycleConfigurationUpdate : NSObject
```

Comment:
This class should have an initializer

Reply:
What initializer other than `init` would you want?

Reply:
An init that takes all of the properties, so it can be one line to create instead of five. I'm not sure if there are any other clear cases of a needed init.